privacyidea/lib/tokens/radiustoken.py

```diff
@@ -45,34 +45,35 @@
 
 import traceback
 import binascii
-from privacyidea.lib.utils import is_true
 from privacyidea.lib.tokenclass import TokenClass, TOKENKIND
-from privacyidea.lib.tokens.remotetoken import RemoteTokenClass
 from privacyidea.api.lib.utils import getParam, ParameterError
 from privacyidea.lib.log import log_with
 from privacyidea.lib.config import get_from_config
 from privacyidea.lib.decorators import check_token_locked
+from privacyidea.lib.policydecorators import challenge_response_allowed
 from privacyidea.lib.radiusserver import get_radius
+from privacyidea.models import Challenge
+from privacyidea.lib.challenge import get_challenges
 
 import pyrad.packet
 from pyrad.client import Client
 from pyrad.dictionary import Dictionary
 from privacyidea.lib import _
 
-
 optional = True
 required = False
 
 log = logging.getLogger(__name__)
 
 
 ###############################################
-class RadiusTokenClass(RemoteTokenClass):
+class RadiusTokenClass(TokenClass):
```

Comment:
I'm not sure I have any reasoning beyond that I ended up not utilizing any of the underpinnings of the RemoteTokenClass so it felt odd to keep the parent class that way.

Reply:
I somehow feel I would like to keep it inherited from the remotetokenclass, since these are supposed to the tokenclasses, that forward the authentication request somewhere else. If there currently are some dependencies not given anymore, there might be some in the future.

(Trying to get back to the review here)